### DISCUSSION TITLE
Issue with Account Verification and Instance Access Post v0.23 Update

### DISCUSSION THREAD

**Discussion by syvixor** [pockethost/pockethost#421] [Q&A]
Since the release of v0.23, I’ve been facing an issue where I’ve lost access to my instances. I tried deleting the existing instances and creating new ones, but the problem persists. Interestingly, I created a new account, verified it, and everything worked fine there. This makes me wonder if the issue is related to the verification of my original account, which was already verified prior to v0.23.

For reference, this issue seems related to the discussion in #413.

![Screenshot](https://github.com/user-attachments/assets/810a3867-84ee-4d2a-83ec-a0c43e627418)

**Comment by benallfree** [comment]
That error means you have to email support or DM me on Discord :)
